activemq-http/src/main/java/org/apache/activemq/transport/ws/jetty9/StompSocket.java

```diff
@@ -46,8 +49,13 @@ public StompSocket(String remoteAddress) {
     @Override
     public void sendToStomp(StompFrame command) throws IOException {
         try {
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
```

Comment:
if that is an issue.. this would be dead slow for the server...

if there's any issues around this.. this fix is not good anyways.

As Tim said.. you need a test.